platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -472,4 +478,100 @@ describe('OHIF Cornerstone Toolbar', () => {
     cy.get('@viewport').click(x2, y2, { force: true });
     cy.get('.measurementItem').should('not.exist');
   });
+
+  it('check if Invert tool will change the colors of the image in the viewport', () => {
+    // Click on More button
+    cy.get('@moreBtn').click();
+    // Verify if overlay is displayed
+    cy.get('.tooltip-toolbar-overlay').should('be.visible');
+
+    // Click on Invert button
+    cy.get('.tooltip-inner')
+      .contains('Invert')
+      .click();
+
+    // check on cornerstone if image was inverted
+    cy.window()
+      .its('cornerstone')
+      .then($cornerstone => {
+        expect($cornerstone.getEnabledElements()[0].viewport.invert).to.be.true;
+      });
+
+    // Visual comparison
+    cy.screenshot();
+    cy.percyCanvasSnapshot('Invert tool');
+  });
+
+  it('check if Rotate tool will change the image orientation in the viewport', () => {
+    //Click on More button
+    cy.get('@moreBtn').click();
+    //Verify if overlay is displayed
+    cy.get('.tooltip-toolbar-overlay')
+      .should('be.visible')
+      .then(() => {
+        //Click on Rotate button
+        cy.get('.tooltip-inner')
+          .contains('Rotate')
+          .click({ force: true });
+      });
+
+    //Check on cornerstone if image was rotated
+    cy.window()
+      .its('cornerstone')
+      .then($cornerstone => {
+        expect($cornerstone.getEnabledElements()[0].viewport.rotation).to.equal(
+          90
+        );
+      });
```

Comment:
Same here. The visual tests allow us to avoid caring about implementation details.